clang/test/CIR/Lowering/union-array.c

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm -fno-clangir-call-conv-lowering %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s --check-prefix=LLVM
```

Comment:
Can you move this test under CodeGen and additionally check the CIR output? (It's a CIRGen change, so we want to see how CIR behaves as well).

Reply:
Done